script.js

```diff
@@ -0,0 +1,38 @@
+const body = document.querySelector("body"); //can search by element, class, or id
+const button = document.querySelector("button");
+const logoText = document.querySelector("#logoText")
+
+
+/* Change background color with press of button*/
+let colors = ['red', 'blue', 'green','yellow', 'pink', 'purple', 'orange' ];
+let lastNum = 0;
+button.addEventListener("click", function(){
```

Comment:
ooo this looks like it would be cool! I wonder, if there isn't a button to click, maybe something else that could have a click handler on it to still change the logo color?